File: 프로그래머스/2/70129. 이진 변환 반복하기/이진 변환 반복하기.py
def solution(s):
    count = 0;
    zero = 0;
    
    while len(s) > 1:
        newS = '';
        for i in s:
            if i == '0': zero += 1;
            if i == '1': newS += '1';
        s = format(len(newS), 'b')
        count += 1

    return [count, zero]

File: 프로그래머스/2/70129. 이진 변환 반복하기/이진 변환 반복하기.js
function solution(s) {
    let count = 0;
    let zero = 0;
    while(s.length > 1) {
        let newS = '';
        for(let i = 0; i < s.length; i++){
            if(s[i] === '0') zero++
            if(s[i] === '1') newS += '1'
        }
        newS = newS.length.toString(2)
        s = newS
        count++
    }
    return [count, zero]
}